dev/.rat-excludes

```diff
@@ -118,3 +118,4 @@ announce.tmpl
 vote.tmpl
 SessionManager.java
 SessionHandler.java
+.python-version
```

Comment:
It seems odd to have this here. What is generating this file? It's definitely not being added by this PR. And other builds are not failing. So I think this deserves an explanation...

Reply:
It's a [pyenv](https://github.com/pyenv/pyenv) metadata file.  I was using pyenv to manage Python versions locally for purposes of running the PySpark tests.  It doesn't really need a license file since it's just a single line indicating the Python version. Nonetheless, it indeed has nothing to do with this change, so I'll remove it.